src/KeyworkerProfile/container.js

```diff
@@ -20,6 +20,8 @@ class KeyworkerProfileContainer extends Component {
   componentWillMount () {
     if (this.props.paramStaffId) {
       this.gotoKeyworkerProfile(this.props.paramStaffId);
+    } else if (this.props.path === "/keyworkerSearch") {
+      this.props.setCurrentPageDispatch(2);
```

Comment:
this is discussion point - its potentially going to get messy with one container and many child components.  i'm considering making each child page have its own container to make routing simpler - i'll do a bit of a spike

Reply:
This looks like exactly the sort of thing that Routes handle - maybe reinforces the idea of using them instead pof page no